src/renderer/components/subscriptions-community/subscriptions-community.js

```diff
@@ -69,19 +74,28 @@ export default defineComponent({
   watch: {
     activeProfile: async function (_) {
       this.isLoading = true
-      this.loadpostsFromCacheSometimes()
+      this.loadPostsFromCacheSometimes()
     },
   },
   mounted: async function () {
     this.isLoading = true
 
-    this.loadpostsFromCacheSometimes()
+    this.loadPostsFromCacheSometimes()
   },
   methods: {
-    loadpostsFromCacheSometimes() {
+    loadPostsFromCacheSometimes() {
       // This method is called on view visible
       if (this.postCacheForAllActiveProfileChannelsPresent) {
```

Comment:
Edit: To more explicitly clarify what I would believe to be the logical behavior here, I think we should have this if statement condition be modified to be `!this.updateFetchSubscriptionsAutomatically || this.postCacheForAllActiveProfileChannelsPresent`. And that I revert my last commit to move the minTimestamp logic outside of this if-statement as was before, as this case would no longer exist as is.